app/assets/javascripts/discourse/components/second-factor-form.js.es6

```diff
@@ -16,17 +16,27 @@ export default Ember.Component.extend({
       : I18n.t("login.second_factor_backup_description");
   },
 
-  @computed("secondFactorMethod")
-  linkText(secondFactorMethod) {
-    return secondFactorMethod === SECOND_FACTOR_METHODS.TOTP
-      ? "login.second_factor_backup"
-      : "login.second_factor";
+  @computed("secondFactorMethod", "isLogin")
+  linkText(secondFactorMethod, isLogin) {
+    if (isLogin) {
+      return secondFactorMethod === SECOND_FACTOR_METHODS.TOTP
+        ? "login.second_factor_backup"
+        : "login.second_factor";
+    } else {
+      return secondFactorMethod === SECOND_FACTOR_METHODS.TOTP
+        ? "user.second_factor_backup.use"
+        : "user.second_factor.use";
+    }
   },
 
   actions: {
     toggleSecondFactorMethod() {
       const secondFactorMethod = this.get("secondFactorMethod");
-      this.set("loginSecondFactor", "");
+      if (this.get("loginSecondFactor")) {
```

Comment:
Wondering what the purpose of checking for if loginSecondFactor before setting it. Is it performance or functional?